src/common/favmulti.js

```diff
@@ -0,0 +1,12 @@
+import logo from "../images.png";
+import Favaccordion from "../common/favaccordian";
+import Multisingle from "../common/multisingle";
+const Favmulti = (props) => {
+  return (
+    <Favaccordion props={props} sdata={props.sdata}>
+      <Multisingle flight={props.data} />
+      <Multisingle flight={props.sdata} />
```

Comment:
why are you using same component twice?

Reply:
for the multiflight  first component for first flight and second for second flight